cancer_ai/chain_models_store.py

```diff
@@ -10,47 +10,31 @@
 class ChainMinerModel(BaseModel):
     """Uniquely identifies a trained model"""
 
-    namespace: str = Field(
-        description="Namespace where the model can be found. ex. Hugging Face username/org."
-    )
-    name: str = Field(description="Name of the model.")
-
-    epoch: int = Field(
-        description="The epoch number to submit as your checkpoint to evaluate e.g. 10"
-    )
-
-    date: datetime.datetime = Field(
```

Comment:
Why we resign from the date? It could prevent from non-unique named models if we pair it with model name